log.txt

```diff
@@ -0,0 +1,66 @@
+INFO:ivadomed.utils:
```

Comment:
I don't think this file is supposed to be track in the repo?

Reply:
I'll revert it, sorry about that! For some reason I didn't see this file indexed after staging my changes. Perhaps we should also open an issue to add and log files to the gitignore file